compiler/rustc_driver/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+// This crate is intentionally empty and a rexport of `rustc_driver_impl` to allow the code in
+// `rustc_driver_impl` to be compiled in parallel with other crates.
+
+#![allow(unused_extern_crates)]
+extern crate rustc_driver_impl;
```

Comment:
This should be unnecessary since `pub use rustc_driver_impl` forces the crate to be loaded, right?

Reply:
It seems so.